feature-group-definitions/highlight.dist.yml

```diff
@@ -0,0 +1,43 @@
+# Generated from: highlight.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: Custom highlights
+description: Custom highlights style arbitrary text ranges, without adding extra elements to the DOM.
+spec: https://drafts.csswg.org/css-highlight-api-1/
+status:
+  baseline: false
+  support:
```

Comment:
Firefox seems to have support for the API in at least Nightly. I see BCD says "preview". I'm not sure what that means though. Should our support mention Firefox?

Reply:
This is the same problem as in https://github.com/web-platform-dx/web-features/issues/955, that the definition of Baseline only considers stable releases, but we should make it visible to reviewers that support might be upcoming.